src/ui/src/builder/journal/BuilderJournalBlockOutput.vue

```diff
@@ -0,0 +1,449 @@
+				</span>
+			</div>
+		</div>
+		<!-- Block Output Tabs -->
+		<div class="output-block__tabs">
+			<WdsTabs
+				:model-value="activeTab"
+				:tabs="blockTabs"
+				variant="bar"
+				@update:model-value="(value) => (activeTab = value)"
+			/>
+		</div>
+
+		<div class="output-block__tab-content">
+			<!-- Error Details Tab -->
+			<div
+				v-if="output.message && activeTab === 'error'"
+				class="output-block__message"
+			>
+				<div class="output-block__message-content">
+					{{ output.message }}
+				</div>
+			</div>
+
+			<!-- Stdout Tab -->
+			<div
+				v-if="output.stdout && activeTab === 'stdout'"
+				class="output-block__log-section"
+			>
+				<pre class="output-block__log-content">{{ output.stdout }}</pre>
+			</div>
+
+			<!-- Logs Tab -->
+			<div
+				v-if="output.logs && activeTab === 'logs'"
+				class="output-block__log-section"
+			>
+				<pre class="output-block__log-content">{{ output.logs }}</pre>
+			</div>
+
+			<!-- Result Tab -->
+			<div
+				v-if="activeTab === 'result'"
+				class="output-block__result-section"
+			>
+				<SharedJsonViewer
+					:data="output.result"
+					is-root
+					is-root-open
+					enable-copy-to-json
+				/>
+			</div>
+		</div>
+	</div>
+</template>
+
+<script setup lang="ts">
+import { computed, PropType, ref, watch } from "vue";
+import WdsIcon from "@/wds/WdsIcon.vue";
+import WdsTabs from "@/wds/WdsTabs.vue";
+import SharedImgWithFallback from "@/components/shared/SharedImgWithFallback.vue";
+import SharedJsonViewer from "@/components/shared/SharedJsonViewer/SharedJsonViewer.vue";
+import { convertAbsolutePathtoFullURL } from "@/utils/url";
+
+type BlockOutput = {
+	result: any;
+	outcome: string;
+	message?: string;
+	stdout?: string;
+	logs?: string;
+	startedAt?: number;
+	executionTimeInSeconds?: number;
+	component?: {
+		type?: string;
+		title?: string;
+	} | null;
+};
+
+const props = defineProps({
+	blockId: { type: String, required: true },
+	output: { type: Object as PropType<BlockOutput>, required: true },
+	index: { type: Number, required: true },
+});
+
+defineEmits<{
+	"go-to-block": [blockId: string];
+}>();
+
+const activeTab = ref<string>("");
+
+// Build tabs based on available data
+const blockTabs = computed(() => {
+	const tabs = [];
+
+	if (props.output.message) {
+		tabs.push({ label: "Error", value: "error" });
+	}
+	if (props.output.stdout) {
+		tabs.push({ label: "Stdout", value: "stdout" });
+	}
+	if (props.output.logs) {
+		tabs.push({ label: "Logs", value: "logs" });
+	}
+	tabs.push({ label: "Result", value: "result" });
+
+	return tabs;
+});
+
+watch(
+	blockTabs,
+	(tabs) => {
+		if (!tabs.length) {
+			activeTab.value = "";
+			return;
+		}
+		if (!tabs.some((tab) => tab.value === activeTab.value)) {
+			activeTab.value = tabs[0].value;
+		}
+	},
+	{ immediate: true },
+);
+
+function formatTimestamp(timestamp: number): string {
+	const date = new Date(timestamp * 1000);
+	return date.toLocaleTimeString("en-US", {
```

Comment:
You can let the browser use the user's local
```suggestion
	return date.toLocaleTimeString(undefined, {
```